methods_to_test.js

```diff
@@ -0,0 +1,69 @@
+const capitalize = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+const reverseString = (string) => {
+    let splitString = string.split("")
+    let reversedArray = splitString.reverse()
+    let joinString = reversedArray.join("")
+
+    return joinString
+}
+
+const analyse = (numbers) => {
+    let sum = numbers.reduce((previous, current) => current += previous) / numbers.length
```

Comment:
Why do we call this "sum" ? We are computing for the average. Let us use a more meaningful variable name.